src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Book.ttf);
+  color: #3d4e61;
+}
+
+.search-image {
+  position: relative;
+}
+
+.search-image::before {
+  content: "";
+  position: absolute;
+  background: url(images/Search.svg) no-repeat;
+  top: 25px;
+  bottom: 0;
+  left: 26px;
+  margin: auto 0;
+  width: 25px;
+  /* height: 25px; */
```

Comment:
remove comments

Reply:
removed comments, but why? can't I use it?